docs(README): Roadmap and maintainer note improvements (#269)
paritytech/txwrapper: README.md

ℹ Run `git push --follow-tags origin master && npm publish` to publish
```

> To publish the new package, just follow the instructions: `git push --follow-tags origin master && npm publish.`

Reviewer: ```suggestion
To publish the new package, run: `git push --follow-tags origin master && npm publish.`
```

You must have access to the @substrate organization on npm to publish.

### Roadmap

> - API revamp to make it more clear how to handle `registry` and `metadataRpc`.

Reviewer: ```suggestion
- API revamp to clarify handling of `registry` and `metadataRpc`.
```

#### Parachain support

> - Factor out utility functions and types (such as `decode`, `getRegistry`, `createMethod` etc) into an sdk for parachain

Reviewer: ```suggestion
- Factor out utility functions and types (such as `decode`, `getRegistry`, `createMethod` etc) into a library for parachain
```

Maybe? To me, an "SDK" is a big ball of libraries, tools and other bits and pieces. This here is more like a single npm package right? :)

Author: Yup, I wasn't sure what word to use.

> teams that will allow them to release and maintain txwrapper libraries specific to their parachains. This could be called

Reviewer: ```suggestion
teams that allows them to release and maintain txwrapper libraries specific to their parachains. This could be called
```

> `@substrate/txwrapper-core`. This lib will allow a parachain team to setup an offline signing lib with unit tests

Reviewer 2: ```suggestion
`@substrate/txwrapper-core`. This lib will allow a parachain team to set up an offline signing lib with unit tests
```

> `@substrate/txwrapper-sdk`. Ideally this sdk will allow a parachain team to setup an offline signing library with unit tests

Reviewer: ```suggestion
`@substrate/txwrapper-core`. This sdk will allow a parachain team to setup an offline signing library with unit tests
```

In Rust-land, it's pretty common to call such libraries "something-core", not sure how well that works for JS.

quickly and painlessly while allowing users access to a `txwrapper` like API across parachains.

> - While core utility will be factored out to its own dependency, dispatchables from generic substrate methods

Reviewer: ```suggestion
- While the core utility will be factored out to its own dependency, dispatchables from generic substrate methods
```

> will be published in a package `@substrate/txwrapper-substrate` and polkadot/kusama specific dispatchables will be available in

Reviewer 2: ```suggestion
will be published in a package `@substrate/txwrapper-substrate` and Polkadot/Kusama specific dispatchables will be available in
```

> `@substrate/txwrapper-polkadot`. (This could be in a mono repo, but separate packages.) Parachains could then create

Reviewer: ```suggestion
`@substrate/txwrapper-polkadot`. (This could be in a mono repo, but separate packages.) Parachains then create
```

> there own txwrapper lib using the sdk and publish it as `@{parachain-name}/txwrapper`.

Reviewer: ```suggestion
their own txwrapper lib using the sdk and publish it as `@{parachain-name}/txwrapper`.
```
